docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -53,7 +55,7 @@ for more details.
 .. _dronecan_support_esc_status:
 
 uavcan.equipment.esc.Status (Data Type ID = 1034)
-##################################################
```

Comment:
You should update this section to point out how what's in the temperature bit changes based on your style of telemetry